pinc/LPage.inc

```diff
@@ -8,37 +8,26 @@ include_once($relPath.'user_is.inc');
 include_once($relPath.'DPage.inc');
 include_once($relPath.'Project.inc');
 include_once($relPath.'page_tally.inc');
-include_once($relPath.'project_continuity.inc');
 include_once($relPath.'maybe_mail.inc');
 include_once($relPath.'daily_page_limit.inc'); // get_dpl_count_for_user_in_round
 
 // XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 
 function get_available_page_array(
```

Comment:
Let's put all of the function args on the same line now that they'll all fit.

Reply:
Done this.